src/docfx.website.themes/default/styles/docfx.css

```diff
@@ -807,6 +807,23 @@ pre > code .line-highlight {
   background-color: #ffffcc;
 }
 
+/* For Embedded Video */
+div.embeddedvideo {
+    padding-top: 56.25%;
+    position: relative;
+    width: 100%;
+}
+
+div.embeddedvideo iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
No, the absolute position (and top/left/right/bottom) keep it in the right spot as the container resizes.